c10/util/Exception.h

```diff
@@ -272,6 +272,12 @@ inline std::string if_empty_then(std::string x, std::string y) {
 #endif
 #define TORCH_CHECK(cond, ...) TORCH_CHECK_WITH(Error, cond, __VA_ARGS__)
 
+#if defined(__CUDACC__) || defined(__HIPCC__)
```

Comment:
There should be a comment here explaining when this macro should be used

Reply:
added